torch/onnx/_internal/onnxruntime.py

```diff
@@ -469,8 +530,24 @@ def is_supported(self, *args):
         if len(args) != len(self.input_value_infos):
             return False
         for arg, value_info in zip(args, self.input_value_infos):
-            if not isinstance(arg, torch.Tensor):
+            if not isinstance(arg, (torch.Tensor, float, int)):
                 return False
+
+            # Check Python scalars such as int, float, and bool.
+            scalar_to_dtype = {
+                float: _TORCH_DTYPE_TO_ONNX_TENSOR_ELEMENT_TYPE[torch.float],
+                int: _TORCH_DTYPE_TO_ONNX_TENSOR_ELEMENT_TYPE[torch.int64],
+                bool: _TORCH_DTYPE_TO_ONNX_TENSOR_ELEMENT_TYPE[torch.bool],
+            }
```

Comment:
Maybe move to or reuse from torch/onnx/_internal/fx/type_utils.py?

Reply:
Sure. Add new function
```py
def from_python_type_to_onnx_tensor_element_type(type: type):
    """
    Converts a Python type to the corresponding ONNX tensor element type.
    For example, `from_python_type_to_onnx_tensor_element_type(float)` returns
    `onnx.TensorProto.FLOAT`.

    Args:
      type (type): The Python type to convert.

    Returns:
      int: The corresponding ONNX tensor element type.

    """
    _PYTHON_TYPE_TO_ONNX_TENSOR_ELEMENT_TYPE = {
      float: onnx.TensorProto.FLOAT,  # type: ignore[attr-defined]
      int: onnx.TensorProto.INT64,  # type: ignore[attr-defined]
      bool: onnx.TensorProto.INT64,  # type: ignore[attr-defined]
    }
    return _PYTHON_TYPE_TO_ONNX_TENSOR_ELEMENT_TYPE.get(type)
```

Note that this is similar to `from_python_type_to_onnx_attribute_type`.